lib/modules/apostrophe-schemas/public/js/user.js

```diff
@@ -761,6 +761,111 @@ apos.define('apostrophe-schemas', {
       }
     };
 
+    self.enableReadOnlyFields = function(data, name, $field, $el, field) {
```

Comment:
ℹ️ logic replicated and adapted from `self.enableShowFields` above.

Reply:
Same question re: making both short wrappers for a shared method that implements the logic when given the appropriate property names.